amoro-format-iceberg/src/test/java/org/apache/amoro/catalog/TableTestBase.java

```diff
@@ -44,14 +45,11 @@ public void setupTable() {
     this.tableMetaStore = MixedFormatCatalogUtil.buildMetaStore(getCatalogMeta());
 
     getUnifiedCatalog().createDatabase(TableTestHelper.TEST_DB_NAME);
-    switch (getTestFormat()) {
-      case MIXED_HIVE:
-      case MIXED_ICEBERG:
-        createMixedFormatTable();
-        break;
-      case ICEBERG:
-        createIcebergFormatTable();
-        break;
+    TableFormat format = getTestFormat();
+    if (TableFormat.MIXED_HIVE.equals(format) || TableFormat.MIXED_ICEBERG.equals(format)) {
```

Comment:
How about replacing with TableFormat#in?

Reply:
fixed